src/app/js/formats/streamgraph/Streamgraph.js

```diff
@@ -481,14 +479,12 @@ class Streamgraph extends PureComponent {
                         )
                         .style('visibility', 'visible');
 
+                    var currentLegendItem;
```

Comment:
it doesn't seem to be used in upper scopes, so I guess you can `const` it inside the loop